packages/@uppy/companion/src/server/provider/zoom/index.js

```diff
@@ -52,11 +52,13 @@ class Zoom extends Provider {
         })
     })
 
+    const userResponse = await userPromise
+
     if (from && to) {
       const queryObj = {
         page_size: PAGE_SIZE,
-        from,
-        to
+        from: moment.tz(from, userResponse.body.timezone || 'UTC').startOf('day').tz('UTC').format('YYYY-MM-DD'),
+        to: moment.tz(to, userResponse.body.timezone || 'UTC').endOf('day').tz('UTC').format('YYYY-MM-DD')
```

Comment:
example: user in Canada (EDT, (UTC -4)) clicks on a folder for a month (eg 08-01 to 08-31), they're really asking for  files for 08-01 0:00 EDT (08-01 04:00 UTC) to 08-31 23:59 EDT (09-01 03:59 UTC). 

so the incoming request from/to is in local time, we convert to their timezone, set it to start/end of day, convert to UTC, and get just the date portion (because Zoom API only accepts date, not date time)

Reply:
makes sense, I've added a comment here https://github.com/transloadit/uppy/pull/2525/files#diff-781d61e346ed162168c7bef9670c505aR55